.github/workflows/pr-build.yml

```diff
@@ -67,6 +67,20 @@ jobs:
         with:
           arguments: build integrationTests --stacktrace -PenableCoverage=true -PlocalDocker=true
 
+      - name: Get current version
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        shell: bash
+        run: |
+          echo ADOT_JAVA_VERSION="$(./gradlew printVersion -q )" >> $GITHUB_ENV
+
+      - name: Build image
+        uses: docker/build-push-action@v4
+        if: ${{ matrix.os == 'ubuntu-latest' }}
+        with:
+          push: false
+          build-args: "ADOT_JAVA_VERSION=${{ env.ADOT_JAVA_VERSION }}"
+          context: .
```

Comment:
Just wondering if we should add the tag here as well?

Reply:
we are not pushing, so no need to providing tags.